source/config/featureFlagEnums.py

```diff
@@ -67,6 +67,24 @@ def __bool__(self):
 		return self == BoolFlag.ENABLED
 
 
+class ParagraphNavigationFlag(DisplayStringEnum):
+	@property
+	def _displayStringLabels(self):
+		return {
+			# Translators: Label for a paragraph style in NVDA settings.
+			self.APPLICATION: _("Handled by application"),
+			# Translators: Label for a paragraph style in NVDA settings.
+			self.SINGLE_LINE_BREAK: _("Single line break"),
+			# Translators: Label for a paragraph style in NVDA settings.
+			self.DOUBLE_LINE_BREAK: _("Double line break")

```

Comment:
the naming scheme here will need to be updated to multi

Reply:
@seanbudd Are we going with multiline, multi-line, or multi line? If I pick one, I'm sure I'll get it wrong. Even dictionaries on the web can't agree about multiline versus multi-line, but I am not seeing it with a space.